app/services/notification_emailer.rb

```diff
@@ -3,10 +3,11 @@
 class NotificationEmailer
 
   class EmailUser
-    attr_reader :notification
+    attr_reader :notification, :no_delay
 
-    def initialize(notification)
+    def initialize(notification, no_delay = false)
```

Comment:
I strongly prefer keyword arguments for things like this. Otherwise at the calling site it's not clear what's happening. 

`EmailUser.new(notificaiton, false)`  - I have to read more code to find out what `false` is
`EmailUser.new(notification, no_delay: true)` - I know we don't want to delay.

Reply:
The reason I did this, is that this initializer is only called from within this file and no_delay is a keyword arg for it's caller, so we will now have `no_delay: no_delay` but I think you're still right.